packages/cli/internal/pkg/cli/context/info_test.go

```diff
@@ -25,3 +29,35 @@ func TestDetail_IsNotEmpty(t *testing.T) {
 	detail := Detail{WesUrl: "amazon.com"}
 	assert.False(t, detail.IsEmpty())
 }
+
+func TestSummary_IsHeadProcessEngine_HeadEnginesShouldReturnTrue(t *testing.T) {
+	for _, engineName := range testHeadEngineNames {
```

Comment:
Please use a parameterized tests. It is better to avoid testing multiple cases in one test.
For more details please read:

https://elliotchance.medium.com/go-data-driven-or-parameterized-tests-cd2b62b21c8a